src/python/WMCore/ReqMgr/Service/Auxiliary.py

```diff
@@ -268,3 +269,12 @@ class PermissionsConig(AuxBaseAPI):
 
     def setName(self):
         self.name = "PERMISSION_BY_REQUEST_TYPE"
+
+
+class CampaignConfig(AuxBaseAPI):
+    """
+    Config which used for Agent
+    """
+
+    def setName(self):
+        self.name = "CAMPAIGN_CONFIG"
```

Comment:
won't it create a default configuration doc (name CAMPAIGN_CONFIG) that will never be used? Since all the other campaign docs are named after the campaign name.

Reply:
when post call is used subname is passed. So CAMPAIGN_CONFIG_[actual_campaign_name] will be uploaded